frontend/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Use this image as the platform to build the web
+FROM node:20-alpine AS dockerbuild
+
+# A small line inside the image to show who made it
+LABEL Developers="Curlybytes"
```

Comment:
Add "DevLeoko" as additional developer due to maintaining of LicenseGate itself.

Reply:
add developer house keeping